src/components/relatedProducts/Carousel.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { FC } from 'react';
+import { useState, useEffect } from 'react';
+import Card from './Card';
+import AddToOutfitCard from './AddToOutfitCard';
+
+interface CarouselProps {
+}
+
+const Carousel: FC<CarouselProps> = ({items, currentProductID, currentProductData, handleCardClick, carouselType, handleAddOutfit, handleRemoveOutfit}) => {
+  console.log('items: ', items)
+
+  // create itemsToDisplay state array with a slice, change range for each type of carousel
+  // create functions for forward and back that update
+  // render arrows conditionally if there are more items before or after
+
+  const [firstItemIndex, setfirstItemIndex] = useState(0)
+  const [lastItemIndex, setLastItemIndex] = useState(4)
+
+  // make exception for last item in shorter outfit array
+  useEffect(() => {
+    if (carouselType === 'outfit') {
+      setLastItemIndex(3)
```

Comment:
you could simplify this by using a ternary in the declaration of lastItemIndex
`const [lastItemIndex, setLastItemIndex] = useState (
  carouselType === 'outfit' ? 3 : 4
)`